src/products/iOS_products/updating_locales.md

```diff
@@ -0,0 +1,28 @@
+# Updating locales for specific projects
+
+These are the instructions to update locales on iOS mobile products, currently living within the [Firefox for iOS project](https://github.com/mozilla-l10n/firefoxios-l10n), and in the [Focus for iOS project](https://github.com/mozilla-l10n/focusios-l10n).
+
+## Prerequisites to adding a locale
+
+First, you will have to check that the locale is available within the iOS system settings. iOS currently supports two tiers of locales, and a locale must be part of one of these two lists in order to ship on Firefox for iOS. Unlike Firefox for Android, there’s no locale switcher included in the app, so it’s possible to ship only locales that are supported by iOS.
```

Comment:
```suggestion
First, you will have to check that the locale is available within the iOS system settings. iOS currently supports two tiers of locales, and a locale must be part of one of these two lists in order to ship on Mozilla products for iOS. Unlike Mozilla products for Android, there’s no locale switcher included in the app, so it’s possible to ship only locales that are supported by iOS.
```

Missed that this would ignore Focus.